### ISSUE TITLE
Serenity 3.3.4 Requirements are not generated on Win OS -> java.nio.file.NoSuchFileException: src\test\resources\features

### ISSUE THREAD

**Issue by martin-cieslar-telia** [serenity-bdd/serenity-core#2959]
Hi,

For Windows-based OS, we receive a lot of `java.nio.file.NoSuchFileException: src\test\resources\features` when the `aggregate` job is started. The report can be shown but without requirements. 
An attempt to click on some requirement in the path to test result ends with 404 Not Found.

plugin: `net.serenity-bdd.serenity-gradle-plugin:3.3.4`
serenity: serenityVersion = '3.3.4'

OS: Windows 10 Enterprise  version: 21H2

Thank you

StackTrace:
```
Failed to load requirements: src\test\resources\features
java.io.UncheckedIOException: src\test\resources\features
	at org.apache.commons.io.FileUtils.listFilesAndDirs(FileUtils.java:2184)
	at net.thucydides.core.files.TheDirectoryStructure.maxDepth(TheDirectoryStructure.java:56)
	at net.thucydides.core.requirements.FileSystemRequirementsTagProvider.getRequirementTypeOf(FileSystemRequirementsTagProvider.java:655)
	at net.thucydides.core.requirements.FileSystemRequirementsTagProvider.requirementFromDirectoryName(FileSystemRequirementsTagProvider.java:643)
	at net.thucydides.core.requirements.FileSystemRequirementsTagProvider.readRequirementFrom(FileSystemRequirementsTagProvider.java:554)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
	at net.thucydides.core.requirements.FileSystemRequirementsTagProvider.capabilitiesAndStoriesIn(FileSystemRequirementsTagProvider.java:187)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.HashMap$KeySpliterator.forEachRemaining(HashMap.java:1621)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at net.thucydides.core.requirements.FileSystemRequirementsTagProvider.getRequirements(FileSystemRequirementsTagProvider.java:173)
	at net.thucydides.core.requirements.FileSystemRequirementsTagProvider.lastRequirementFrom(FileSystemRequirementsTagProvider.java:455)
	at net.thucydides.core.requirements.FileSystemRequirementsTagProvider.parentRequirementFromPackagePath(FileSystemRequirementsTagProvider.java:348)
	at net.thucydides.core.requirements.FileSystemRequirementsTagProvider.getParentRequirementOf(FileSystemRequirementsTagProvider.java:333)
	at net.thucydides.core.requirements.ParentRequirementsProvided.forOutcome(ParentRequirementsProvided.java:26)
	at net.thucydides.core.requirements.BaseRequirementsService.getParentRequirementOf(BaseRequirementsService.java:139)
	at net.thucydides.core.requirements.BaseRequirementsService.getParentRequirementFor(BaseRequirementsService.java:49)
	at net.thucydides.core.requirements.reports.ScenarioOutcomes.outcomeFrom(ScenarioOutcomes.java:77)
	at net.thucydides.core.requirements.reports.ScenarioOutcomes.outcomeFrom(ScenarioOutcomes.java:59)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at net.thucydides.core.requirements.reports.ScenarioOutcomes.from(ScenarioOutcomes.java:30)
	at net.thucydides.core.reports.html.RequirementsOverviewReportingTask.generateReports(RequirementsOverviewReportingTask.java:159)
	at net.thucydides.core.reports.html.ReportExecutor.call(ReportExecutor.java:33)
	at net.thucydides.core.reports.html.ReportExecutor.call(ReportExecutor.java:12)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.nio.file.NoSuchFileException: src\test\resources\features
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileAttributeViews$BasicWithKey.readAttributes(WindowsFileAttributeViews.java:164)
	at java.base/sun.nio.fs.WindowsFileAttributeViews$BasicWithKey.readAttributes(WindowsFileAttributeViews.java:153)
	at java.base/java.nio.file.FileTreeWalker.getAttributes(FileTreeWalker.java:225)
	at java.base/java.nio.file.FileTreeWalker.visit(FileTreeWalker.java:282)
	at java.base/java.nio.file.FileTreeWalker.walk(FileTreeWalker.java:328)
	at java.base/java.nio.file.Files.walkFileTree(Files.java:2792)
	at org.apache.commons.io.FileUtils.listAccumulate(FileUtils.java:2076)
	at org.apache.commons.io.FileUtils.listFilesAndDirs(FileUtils.java:2179)
	... 44 more

```

**Comment by martin-cieslar-telia** [comment]
I did some configuration tries.
Our original setup was: gradle: 7.0.2 and serenity plugin: 3.1.10

<table>
<tr><th colspan=5>The aggregate task is working:</th></tr>
<tr><td>gradle\serenity-plugin</td><td>3.1.10</td><td>3.2.0</td><td>3.2.1</td><td>3.3.4</td></tr>
<tr><td>7.0.2</td><td>YES </td><td>YES </td><td>NO </td><td>NO </td></tr>
<tr><td>7.3</td><td>YES </td><td>YES </td><td>NO </td><td>NO </td></tr>
<tr><td>7.5.1</td><td>YES </td><td>YES </td><td>NO </td><td>NO </td></tr>
</table>

So it looks like the problem is in the serenity plugin.

I can try to look at it maybe next week.

**Comment by wakaleo** [comment]
I did a quick try rolling back to version 3.2.0, but got the same results.

**Comment by wakaleo** [comment]
The problem appears to be that Gradle on Windows (but not on Linux) defines the working directory as the place the gradle binary is, rather than the current project, which means relative paths don't work.

**Comment by wakaleo** [comment]
Possibly fixed in 3.3.10

**Comment by martin-cieslar-telia** [comment]
Hi, 
thank you, I can confirm, that the plugin version 3.3.10 works OK on windows as well as on unix system.
